contracts/Portfolio.sol

```diff
@@ -222,6 +229,11 @@ abstract contract PortfolioVirtual is Objective {
 
     /**
      * @dev Increases virtual reserves and liquidity. Debits `msg.sender`.
+     * @param deltaLiquidity Quantity of liquidity to mint in WAD units.
+     * @param maxDeltaAsset Maximum quantity of asset tokens paid in WAD units.
+     * @param maxDeltaQuote Maximum quantity of quote tokens paid in WAD units.
+     * @return deltaAsset Real quantity of `asset` tokens paid to pool, in native token decimals.
+     * @return deltaQuote Real quantity of `quote` tokens paid to pool, in native token decimals.
```

Comment:
Maybe the reasoning here will become more apparent as I get into the code further, but at first blush having arguments in both WAD and native decimals seems like the sort of pattern we want to be eliminating/avoiding, not introducing.